### PR TITLE
added trace2html

### PR DESCRIPTION
**Pull request by brentvatne**
On [React Native](https://github.com/facebook/react-native) we will be using `trace2html` to visualize the results from our profiler - it would be great to be able to just have users install via Homebrew, even though it's fairly simple.

### CODE REVIEW

File: Library/Formula/trace2html.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Trace2html < Formula
+  desc "Utility from Google Trace Viewer to convert JSON traces to HTML"
+  homepage "https://github.com/google/trace-viewer"
+  url "https://github.com/google/trace-viewer/archive/a4c4894801935627ff412c3cc6c3ddae96019cc5.zip"
+  sha256 "b9fc71e6f1a36f91ca09c1ea26ae2c38e42d3183b6470e603d1db8b1ee6e8316"
+
+  depends_on "Python"
```
Comment (DomT4) [inline Library/Formula/trace2html.rb]: Don't hardcode Homebrew's Python. We use the system Python on OS X versions that support >2.7. You can invoke that dependency with `depends_on :python if MacOS.version <= :snow_leopard`.

File: Library/Formula/trace2html.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  def install
+    cp_r(buildpath, libexec)
```
Comment (DomT4) [inline Library/Formula/trace2html.rb]: `libexec.install Dir["*"]`

**Comment by DomT4** [comment]
Couple of Audit nits as well:

```
==> audit problems
trace2html:
 * Stable: Use GitHub tarballs rather than zipballs (url is https://github.com/google/trace-viewer/archive/a4c4894801935627ff412c3cc6c3ddae96019cc5.zip).
 * Use `system "#{bin}/trace2html", "test.json"` instead of `system "#{bin}/trace2html test.json"` 
```

**Comment by brentvatne** [comment]
Thanks @DomT4! Fixed! We're releasing a new version tomorrow and it would be great to include this install step in the release notes, but if it's not possible to merge before then, that's totally understandable.

**Comment by DomT4** [comment]
We'll need an actual version or git tag from upstream to include it, rather than just a commit tarball - But if we can get that done it's not impossible to get things in relatively quickly. The tool is popular enough and supported enough to meet our requirements.

**Comment by brentvatne** [comment]
Thanks again @DomT4 - I created an issue on the trace-viewer repo, as you can see above, and referenced this.

**Comment by brentvatne** [comment]
@DomT4 - The trace-viewer team was very responsive and have already gone ahead and added a tagged release. This pull request has been updated accordingly :smile:

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `trace2html 2015-06-23 (new formula)`.

**Comment by DomT4** [comment]
@brentvatne Thanks for the responsiveness on this, both from you and upstream! :beers:

**Comment by brentvatne** [comment]
Thanks a lot guys! Sorry about that @mikemcquaid - I'll do that in the future

**Comment by paramaggarwal** [comment]
Just landed here looking for an easy way to install `trace2html` for use in React Native - thanks @brentvatne and @mikemcquaid!

**Pull request closed without merging**